### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by binaryoverload**
I'm working on revamping the README to contain more useful information like contributing, project info etc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+    <a href="https://github.com/CascadeBot/CascadeBot/pulls"><img src="https://img.shields.io/github/issues-pr/cascadebot/cascadebot.svg?color=purple&style=flat-square" alt="GitHub Pull Requests"></a>
+    <a href="https://github.com/CascadeBot/CascadeBot/network/members"><img src="https://img.shields.io/github/forks/cascadebot/cascadebot.svg?style=flat-square" alt="GitHub Forks"></a>
+    <a href="https://github.com/CascadeBot/CascadeBot/stargazers"><img src="https://img.shields.io/github/stars/cascadebot/cascadebot.svg?style=flat-square" alt="GitHub Stars"></a>
+    <a href="https://cascadebot.org"><img src="https://img.shields.io/website/https/cascadebot.org.svg?down_color=red&down_message=offline&style=flat-square&up_message=online" alt="Website"></a>
+    <br><br>
+    <a href="https://discord.gg/P23GZFB"><img src="https://discordapp.com/api/guilds/488394590458478602/widget.png?style=banner2" alt="Discord Server"></a>
+</p>
 
 CascadeBot runs on JDK 11 to have the latest features available to provide the best user experience.
```
Comment (JamesHawkinss) [inline README.md]: We're really going to need more information here. Even if it's just a list of developers or the plugin's main features, they should be listed both on the website and here.

Comment (JamesHawkinss) [reply README.md]: Oh I didn't see the PR comment, I'm retarded don't worry

Comment (binaryoverload) [reply README.md]: 👏

Comment (JamesHawkinss) [reply README.md]: Good so far :p

**Pull request merged**